compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -286,9 +286,9 @@ contextForFile file = do
         { ctxModules = Map.fromList encodedModules
         , ctxPackages = [(dalfPackageId pkg, dalfPackageBytes pkg) | pkg <- Map.elems pkgMap]
         , ctxDamlLfVersion = lfVersion
-        , ctxLightValidation = case envScenarioValidation of
-              ScenarioValidationFull -> SS.LightValidation False
-              ScenarioValidationLight -> SS.LightValidation True
+        , ctxSkipValidation = case envScenarioValidation of
+            ScenarioValidation True -> SS.SkipValidation False
```

Comment:
Having `ScenarioValidation True` map to `SkipValidation False` is extremely confusing. I would just use the same newtype for both or at least have the same boolean flags underneath instead of inverting them.